.pre-commit-config.yaml

```diff
@@ -1,63 +1,67 @@
 repos:
-- hooks:
-  - id: black
-    language_version: python3
-  repo: https://github.com/ambv/black
-  rev: 19.10b0
--   repo: https://github.com/asottile/seed-isort-config
```

Comment:
Formatted to 2 spaces indentation. Please review the PR by hiding whitespace changes.